app/labor/rate_limit_checker.rb

```diff
@@ -40,19 +27,38 @@ def track_image_uploads
 
   def limit_by_email_recipient_address(address)
     # This is related to the recipient, not the "user" initiator, like in action.
-    EmailMessage.where(to: address).
-      where("sent_at > ?", 2.minutes.ago).size > 5
+    EmailMessage.where(to: address).where("sent_at > ?", 2.minutes.ago).count >
```

Comment:
Let's switch back this to `.size`, it doesn't hurt, and if email messages are not loaded as they would be it's going to call `.count` anyway